docs/source.rst

```diff
@@ -4,73 +4,127 @@ Source Guide
 Choosing the Right Location
 ---------------------------
 
-When national security is involved, we suggest you buy a new computer,
-a USB key and pay cash. In any case you must then find a busy
-coworking place or cyber cafe you don't regularly go to and sit at a
-place with your back to a wall to avoid cameras capturing information
-on your screen or keystrokes.
+When national security is involved, we suggest you buy a new computer and a
+USB flash drive, using cash. In any case you must then find a busy coworking
+place or cyber cafe you don't regularly go to and sit at a place with your back
+to a wall to avoid cameras capturing information on your screen or keystrokes.
 
 Get the Tor Browser
 -------------------
 
-Each SecureDrop instance has a publicly available *Source Interface*: a website where sources can create anonymous accounts, submit files and messages, and check back for replies.
+Each SecureDrop instance has a publicly available *Source Interface:* a website
+where sources can create anonymous accounts, submit files and messages, and
+check back for replies.
+
+Each *Source Interface* is only available as an onion service, which is a
+special type of website with an address ending in ".onion" that is only
+accessible through Tor. Tor is an anonymizing network that makes it difficult
+for anybody observing the network to associate a user's identity (e.g., their
+computer's IP address) with their activity (e.g., uploading information to
+SecureDrop).
+
+The easiest and most secure way to use Tor is to download the Tor Browser from
+the `Tor Project website`_. The Tor Browser is a modified version of the Firefox
+web browser. It was designed to protect your security and anonymity while
+using Tor. If there is a chance that downloading the Tor Browser raises
+suspicion, you have a few alternatives, for example:
+
+* If your mail provider is less likely to be monitored, you can send a mail to
+  gettor@torproject.org with the text "linux", "windows" or "osx" in the body
+  (for your preferred operating system) and a bot will answer with instructions.
+* If you routinely use Git, you can use GitLab to
+  `download the Tor Browser <https://gitlab.com/thetorproject/gettorbrowser/tree/torbrowser-releases>`__.
+
+While using the Tor Browser on your personal computer helps hide your activity
```

Comment:
OK, added a note to that effect in  53e978e3a3ee5bfbdb81c2f144c00b4f60edb73b. I agree this is an important -- potentially life-altering -- warning to have, but am also worried about people completely misunderstanding it. What do you think about this wording, does it strike the right balance?

Reply:
OK—is this, then, validation @emkll that we should take more aggressive measures at the beginning of the Source UI to get users to change their Tor browser's security settings? Relevant to this one edit—let's be sure to give them clear guidance that WILL assuredly protect them, and let them know that, if indeed it is decided to state that in its default config Tor will not conceal their use of Tor. It's ok to make extreme statements, but only if we offer direct remedies for users to take to protect themselves.

Reply:
In this (very specific case), I don't think the Tor browser settings would change much to the overall risk. This is a subset of the first contact problem, where we need to bring more awareness about Tor and its properties more generally. This might, in the future, be useful language to include in landing pages, but I think that for now a disclaimer is warranted and also the best we can do. The wording seems quite good to me in 53e978e